pycbc/pnutils.py

```diff
@@ -655,7 +667,7 @@ def dtdv_func(v):
         return 1. + v * v * (dtdv2 + v * (dtdv3 \
                 + v * (dtdv4
                 + v * (dtdv5
-                + v * ((dtdv6 + dtdv6log*3.*log(v))
+                + v * ((dtdv6 + dtdv6log*3.*numpy.log(v))
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent
* [W503] line break before binary operator

<!-- Comment by [SideCI](https://sideci.com) -->